spec/requests/users_controller_spec.rb

```diff
@@ -4073,6 +4073,18 @@ def create_and_like_post(likee, liker)
             .to_not include(private_group.name)
         end
 
+        it "allows plugin to filter by additional scope" do
+          Group.expects(:custom_scope).never
```

Comment:
Is it possible to avoid using mocks here? Is there some state that we can test for to ensure that the method is not called?